lib/routes/shisu/en.ts

```diff
@@ -0,0 +1,61 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { got } from 'got';
```

Comment:
`got` is a dev dependency and is not available in production environment.

Reply:
There are still many routes using got, so I tried it, but ofetch doesn't seem to support rejectUnauthorized?. Now using puppeteer.

Reply:
Those routes are using `@/utils/got` which is wrapper of ofetch.

Reply:
Yep, and since it doesn't seem to support the insecure rejectUnauthorized: false, I have to use puppeteer instead.